owl/EASE-SAY.owl

```diff
@@ -123,6 +277,83 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#ESTSchemaTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ESTSchemaTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#ImageSchemaTheory"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#EntityRole"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A most general image schema, denoting the existence of some entity.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#EntityRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#EntityRole">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies"/>
+                <owl:someValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Denotes an existing entity, included in ESTSchemas.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#ExecutableSchematicTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ExecutableSchematicTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#SchematicTheory"/>
+        <rdfs:comment>Also known as &quot;executing schemas&quot; or &quot;x-schemas&quot;, these were defined by Bergen and Chang in their work &quot;Embodied Construction Grammar in Simulation-Based Language Understanding&quot; as:
```

Comment:
Best would be to have one explanatory sentence what this is, then make the references.

Reply:
Not sure I follow. Is the exposition somehow unclear? We say where the notion comes from, ie. its original conceptualization that is relevant for us, and then modify that notion to suit SOMA needs.

Reply:
no, what I meant is that in this comment, you first provide an aka sentence before stating what the concept is. My suggestion is to first say what this concept is and afterwards refer to x-schema and co.

Reply:
But that's fairly common to do in dictionary definitions, e.g. the wikipedia article for first order logic (https://en.wikipedia.org/wiki/First-order_logic).

Reply:
well it is not critical, I agree. But if you look at comments in DUL.owl, you will see they almost all follow the same pattern. Anyway, it can stay like this